### PR TITLE
Ammo stuff

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Is this one of the worst ways anyone ever done a PR, yes, does it matter, no not really the git history already looks sus as is.

Rearranges orders of items and fills out certain blanks of the "marshal" roundstart disks H&S Non Lethal and SA Lethal / Pistol/Rifle disks and exotic (which they don't get), instead of clumps of shells now contain shotgun tubes.
Shotgun tubes were made bit less expensive metal wise but now require bit of plastic to make, cost 2 points off disks except exotic, shotgun magazines (SBAW) were reduced back to 8 rounds (if you use realism argument largest shell magazines are 8 rounders) with drums being back to 12 (smallest shotgun drums RL too) on flipside shotgun box packets off disks now only cost 2 instead of 5 points to print, made more expensive metal wise to compensate for that fact alongside the fact they hold 50 instead of 35.
Rifle (7.62) for longest time had their large boxes cost less than the small (15 v 20 and 1 cardboard) now it's 30 like 6.5 and 8.6 has.
SLAP and AP from 8.6 and 7.62 naming removed in favor of calling everything unified HV (high-velocity) to make it less confusing for everyone in the room on what you are actually talking about.
Fixed the inconsistent spelling in carbine magazines.
	
<hr>
</details>

## Changelog
:cl:
tweak: H&S LTL disk, SA Lethal / Pistol / Rifle / Exotic disk contents
balance: Ammo
balance: Shotgun magazines
spellcheck: Carbine magazines
spellcheck: Inconsistent HV naming
/:cl:

### CODE REVIEW

**Comment by Lamasmaster** [comment]
MFW I didn't use the updated file fr fr we love local repos

**Comment by cdb-is-not-good** [comment]
Gonna wanna give this a looking over myself in the next few days before it gets TM'd

**Comment by cdb-is-not-good** [comment]
Thank you kitten

**Comment by Lamasmaster** [comment]
Okay seemingly fixed now, we love using outdated files on main

File: code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm [hunk 1]
```diff
@@ -24,9 +24,7 @@
 		/datum/design/autolathe/ammo/rifle_short_rubber,
 		/datum/design/autolathe/ammo/hrifle_rubber,
 		//shells
-		/datum/design/autolathe/ammo/shotgun_blanks = 0,
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:27]: why are you removing blanks/illumination shells?
I dislike that

**Comment by Trilbyspaceclone** [comment]
ammocan_serb seems to only been removed?
Not sure if thats intented

**Comment by Lamasmaster** [comment]
> ammocan_serb seems to only been removed? Not sure if thats intented

Forgot to add this to the sec misc disk despite putting comments up for it that's my b

File: code/modules/projectiles/ammunition/boxes.dm [hunk 2]
```diff
@@ -707,7 +707,7 @@
 	caliber = CAL_SHOTGUN
 	w_class = ITEM_SIZE_NORMAL //so we dont magiclly shrink
 	ammo_type = /obj/item/ammo_casing/shotgun
-	max_ammo = 35
+	max_ammo = 50
```
[the remark this answers is not in the document]
Comment (Lamasmaster) [reply code/modules/projectiles/ammunition/boxes.dm:710]: > I heavily dislike this thats way to many shells can we go with 40?
Prob could, the shells are still cheap to make and carry

[the remark this answers is not in the document]
Comment (Trilbyspaceclone) [reply code/modules/projectiles/ammunition/boxes.dm:710]: Your lowering the points to print and then increasing the ammo amount
so its both ends buffed is all, seems a LOT

**Comment by Trilbyspaceclone** [comment]
Please keep blanks/luma in HS LTL disk

File: code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm (hunk 1 above)
Comment (Lamasmaster) [reply code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:27]: Neither of those fit the theme of the disk, being LTL, you could argue illumination shells can blind people but that's not really what it's meant to do, blanks are well blanks. Moved it to the rifle ammo disk for blanks and illumination next to flare shells on misc

**Comment by Lamasmaster** [comment]
Forgor about mech ammo should all be updated to the newer names alongside the guns in RND. Shell costs reduced a little bit so they more on par with 7.62 than 8.6

**Comment by Lamasmaster** [comment]
Increased cost of HV ammunition by 1.5 in ammunition fabricator to see if this makes it less common or not, 40mm full print boxes added to it too to make it less obnoxious to print them out (Cost is for full box of 6 like in armory).
Added 10mm submachine guns to the marshal disks (they get a 10mm SMG so does blackshield) and vendors so that people can buy them that way too.

**Comment by Lamasmaster** [comment]
Yeah think I'm happy with how it is as is, not going to make any more changes unless someone spots something dumb

**Comment by Lamasmaster** [comment]
image

**Comment by cdb-is-not-good** [comment]
Is there any reason we can't merge this now

**Pull request merged**